src/components/common/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -65,6 +65,8 @@ const MSG = defineMessages({
 interface Props extends ActionDialogProps {
   verifiedUsers: MemberUser[];
   // showWhitelistWarning: boolean;
+  handleIsForceChange: SetStateFn;
```

Comment:
Small detail but I like the convention where you use the `handle` prefix for handlers that are internal (defined inside) for a particular component and the `on` prefix for any external handlers passed as props (or `set`, since this is a state setter)